src/util/api/api.js

```diff
@@ -21,21 +21,32 @@ export const apiGetLikeButtonMyStatus = (id, data) => {
   const {
     referrer = '',
     isCookieSupport,
+    iscnId = '',
   } = data;
   const cookieParam = isCookieSupport !== undefined ? `&cookie_support=${isCookieSupport ? 1 : 0}` : '';
   return axios.get(
-    `${LIKECOIN_MISC_API_BASE}/like/likebutton/${id}/self?referrer=${encodeURIComponent(referrer)}${cookieParam}&show_count=0`,
+    id === 'iscn'
+      ? `${LIKECOIN_API}/like/likebutton/iscn/self?iscn_id=${encodeURIComponent(iscnId)}${cookieParam}&show_count=0`
+      : `${LIKECOIN_MISC_API_BASE}/like/likebutton/${id}/self?referrer=${encodeURIComponent(referrer)}${cookieParam}&show_count=0`,
```

Comment:
Why iscn uses `LIKECOIN_API` but ${id} uses `LIKECOIN_MISC_API_BASE`? Shouldn't they be the same endpoint?